python/hsml/core/dataset_api.py

```diff
@@ -309,6 +307,22 @@ def chmod(self, remote_path, permissions):
             "PUT", path_params, headers=headers, query_params=query_params
         )
 
+    def exists(self, path: str):
+        """Check if a file exists in the Hopsworks Filesystem.
+
+        # Arguments
+            path: path to check
+        # Returns
+            `bool`: True if exists, otherwise False
+        # Raises
+            `RestAPIError`: If unable to check existence for the path
+        """
+        try:
+            self._get(path)
```

Comment:
Will _get(path) always throw an exception if the path doesn't exist?
Otherwise, we could do `return self._get(path) is not None` inside the try/except to make sure

Reply:
yes it does throw not found exception